openc3/lib/openc3/config/config_parser.rb

```diff
@@ -449,16 +449,20 @@ def parse_errors(errors)
       return if errors.empty?
       message = ''
       errors.each do |error|
+        # Once we have a message we want to ignore errors about bad targets
+        # because the real error is probably within the target
+        next if message != '' && error.message.include?('Unknown keyword and parameters for Target')
+
         if error.is_a? OpenC3::ConfigParser::Error
           message += "\n#{File.basename(error.filename)}:#{error.line_number}: #{error.line}"
           message += "\nError: #{error.message}"
           message += "\nUsage: #{error.usage}" unless error.usage.empty?
-          message += "\nBacktrace:"
```

Comment:
This backtrace is really useful when there are bugs in COSMOS itself.  Maybe just comment out these lines or make them debug logs?

Reply:
Or have the backtrace on only the first error?

Reply:
We never want the backtrace on a ConfigParser::Error because those are special errors we're creating.